SBAT.md

```diff
@@ -140,29 +140,39 @@ value associated with the data currently stored in that variable.
 
 #### Generation-Based Revocation Scenarios
 
-Products (**not** vendors, a vendor can have multiple products or even pass a
+**Products** (**not** vendors, a vendor can have multiple products or even pass a
 product from one vendor to another over time) are assigned a name. Product
 names can specify a specific version or refer to the entire product family. For
 example mydistro and mydistro,12.
 
-Components that are used as a link in the UEFI Secure Boot chain of trust are
+**Components** that are used as a link in the UEFI Secure Boot chain of trust are
 assigned names. Examples of components are shim, GRUB, kernel, hypervisors, etc.
 
+Below is an example of a product and vendor, both in the same `sbat.csv` file. SBAT
+is a vendor, as it is a link in the UEFI Secure Boot chain of trust.
```

Comment:
I think you meant to say SBAT acts like a component rather than a vendor. And yes, the sbat version basically does.